jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGeneratorTest.java

```diff
@@ -228,40 +227,31 @@ void slimServerTrueDist(@Mocked final JavaProject project) {
             .satisfies(o -> assertThat(o).last()
                 .isEqualTo("-Dmaven.repo.local=/deployments/myrepo"));
     }
-    
-    private GeneratorContext contextForSlimServer(JavaProject project, Map<String, Object> bootableJarConfig, Path dir) {
-      Plugin plugin = Plugin.builder().artifactId("wildfly-jar-maven-plugin").groupId("org.wildfly.plugins")
-          .configuration(bootableJarConfig).build();
-      List<Plugin> lst = new ArrayList<>();
-      lst.add(plugin);
 
-      if (dir == null) {
-        new Expectations() {{
-            project.getPlugins();
-            result = lst;
-            context.getProject();
-            result = project;
-          }};
-      } else {
-        new Expectations() {{
-            project.getPlugins();
-            result = lst;
-            project.getBaseDirectory();
-            result = dir.toFile();
-            context.getProject();
-            result = project;
-          }};
-      }
-      return context;
+    private GeneratorContext contextForSlimServer(Map<String, Object> bootableJarconfig, Path dir) {
+        Plugin plugin
+                = Plugin.builder().artifactId("wildfly-jar-maven-plugin").
+                groupId("org.wildfly.plugins").configuration(bootableJarconfig).build();
+        List<Plugin> lst = new ArrayList<>();
+        lst.add(plugin);
+        if (dir == null) {
+            when(project.getPlugins()).thenReturn(lst);
+            when(context.getProject()).thenReturn(project);
+        } else {
+            when(project.getPlugins()).thenReturn(lst);
+            when(project.getBaseDirectory()).thenReturn(dir.toFile());
+            when(context.getProject()).thenReturn(project);
+        }
+        return context;
     }
 
-    private GeneratorContext createGeneratorContext() {
-        new Expectations() {{
-            context.getProject(); result = project;
-            project.getOutputDirectory(); result = temporaryFolder.toFile();
-            project.getPlugins(); result = Collections.emptyList(); minTimes = 0;
-            project.getVersion(); result = "1.0.0"; minTimes = 0;
-        }};
+
+    private GeneratorContext createGeneratorContext() throws IOException {
+        when(context.getProject()).thenReturn(project);
+        String tempDir = Files.createTempDirectory("wildfly-jar-test-project").toFile().getAbsolutePath();
+        when(project.getOutputDirectory()).thenReturn(new File(tempDir));
```

Comment:
Why was this change required? Was `temporaryFolder.toFile()` causing problems with mockito?